### PR TITLE
feat(suite): improve skip backup and PIN modals copy

### LINKED ISSUE #14712: Skip PIN & Backup copy improvements
Please improve the body text copy for the Skip Backup and Skip PIN modals. It can't be done in Crowdin, as there it's just one translation key `TR_DO_YOU_REALLY_WANT_TO_SKIP` 

We need to create two new keys, separate ones for skipping Backup and skipping PIN. Let's not recycle the original one to avoid mistakes in translations or similar.

New copy for skip Backup:
`A wallet backup lets you recover your funds if your Trezor is lost, stolen, or damaged. Without a backup, you could lose access to your crypto permanently.`

New copy for skip PIN: 
`A device PIN prevents unauthorized access to your Trezor. Without it, anyone with your device can access your funds.`

Image
Image

### PR DESCRIPTION
**Pull request by seibei-iguchi**
## Screenshots:

Screenshot 2024-10-08 at 9 33 31
Screenshot 2024-10-08 at 9 33 53

### CODE REVIEW

**Comment by MiroslavProchazka** [comment]
cc @Hermez-cz

File: packages/suite/src/components/onboarding/SkipStepConfirmation.tsx [hunk 1]
```diff
@@ -1,38 +1,55 @@
...
+const getModalContent = (activeStepId: AnyStepId): SkipModalContent => {
     switch (activeStepId) {
         case STEP.ID_SECURITY_STEP:
         case STEP.ID_BACKUP_STEP:
-            text = <Translation id="TR_SKIP_BACKUP" />;
-            nextStep = STEP.ID_SET_PIN_STEP;
-            break;
+            return {
+                heading: <Translation id="TR_SKIP_BACKUP" />,
+                cta: <Translation id="TR_SKIP_BACKUP" />,
```
Comment (jvaclavik) [inline packages/suite/src/components/onboarding/SkipStepConfirmation.tsx]: I believe this shouldn't be call to action. WDYT?

Screenshot 2024-10-08 at 11 43 52

File: packages/suite/src/support/messages.ts [hunk 2]
```diff
@@ -2653,10 +2653,20 @@ export default defineMessages({
...
     TR_SKIP_BACKUP: {
-        defaultMessage: 'Skip Backup',
+        defaultMessage: 'Skip backup',
         id: 'TR_SKIP_BACKUP',
     },
+    TR_SKIP_BACKUP_DESCRIPTION: {
+        defaultMessage:
+            'A wallet backup lets you recover your funds if your Trezor is lost, stolen, or damaged. Without a backup, you could lose access to your crypto permanently.',
+        id: 'TR_SKIP_BACKUP_DESCRIPTION',
+    },
```
Comment (jvaclavik) [inline packages/suite/src/support/messages.ts:2669]: Did you update it in crowdin?

Comment (seibei-iguchi) [reply packages/suite/src/support/messages.ts:2669]: These are new IDs.. I don't have to add them to Crowdin, right?

File: packages/suite/src/components/onboarding/SkipStepConfirmation.tsx (hunk 1 above)
Comment (seibei-iguchi) [reply packages/suite/src/components/onboarding/SkipStepConfirmation.tsx]: I know, I just didn't want to think too much about the naming:D

File: packages/suite/src/support/messages.ts (hunk 2 above)
Comment (jvaclavik) [reply packages/suite/src/support/messages.ts:2669]: True, but you need to do it for `TR_SKIP_BACKUP`

File: packages/suite/src/components/onboarding/SkipStepConfirmation.tsx (hunk 1 above)
Comment (jvaclavik) [reply packages/suite/src/components/onboarding/SkipStepConfirmation.tsx]: What about `secondaryButtonText`?

_disclaimer: I know it's tertiary, but we will refactor all tertiary ones to secondary ones_

Comment (Hermez-cz) [reply packages/suite/src/components/onboarding/SkipStepConfirmation.tsx]: Yes pls! Thanks guys!

**Review by jvaclavik — APPROVED** [review]

**Pull request merged**